src/docs/Foundations/Colors/index.mdx

```diff
@@ -1,48 +1,48 @@
 ---
 title: 'Colors'
 order: 3
-description: 'Mozaic offers an adaptive and a scalable color system. The color palette has been divided in three categories: primary colors, secondary colors and greys.'
+description: 'Mozaic offers an adaptive and scalable color system. The color palette has been divided into three categories: primary colors, secondary colors, and greys.'
```

Comment:
~~Mozaic offers an adaptive and scalable color system. The color palette has been divided into three categories: primary colors, secondary colors, and greys.~~

Mozaic offers an adaptive and scalable color system. The color palette has been divided into four categories: Primary colors, Secondary colors, Greys and Status colors.